__test__/update.test.js

```diff
@@ -0,0 +1,34 @@
+import addTask from '../src/modules/add.js';
+import displayTasks from '../src/modules/display.js';
```

Comment:
- [ ]  Please add tests on this project to mock the localStorage updates

Reply:
Done! 😄

https://github.com/Camilovelag/Todo_list/blob/94ff98537b3c093ba414e2318aa332c0c7b877ec/__mocks__/mockLocalStorage.js#L1-L9